doc/introduction/measurements.rst

```diff
@@ -162,14 +164,15 @@ it can sometimes be convenient to execute the same QNode with differing
 number of shots.
 
 For simulators like ``default.qubit``, finite shots will be simulated if
-we set ``analytic=False`` in the device.
+we set ``shots=None`` in the device.
```

Comment:
Ummmm? I think this is a mistake here.  Did you mean something like:

```suggestion
we set ``shots`` to a positive integer.
```

Reply:
Good catch!!